dev.md

```diff
@@ -6,6 +6,23 @@ overview tbd.
 
 tbd.
 
+**Setup a local dev environment**
+
+You can use the docker-compose [stack](stack) to start all the necessary infrastructure components (like e.g. kafka and minio).
+
+**Applications and profiles**
+
+To start the application you can use the profiles:
+
+* `local` for local development settings
+* `streaming` to enable event streaming to kafka
+
+Additionally, you can use the following profiles to disable a few features to simplify development:
+
+* `no-security` to disable spring security
```

Comment:
There is also a list of profiles in engine-service/README. If this is outdated, please delete it.